index.js

```diff
@@ -1,9 +1,17 @@
 'use strict'
 
 const fp = require('fastify-plugin')
-const pg = require('pg')
+let pg = require('pg')
 
 function fastifyPostgres (fastify, options, next) {
+  if (options.native) {
```

Comment:
Since we are passing the options object to `Pool`, it's better that we remove the `native` key from the object.
```js
if (options.native) {
   delete options.native
   ...
```